src/config.c

```diff
@@ -3162,6 +3162,8 @@ standardConfig static_configs[] = {
     createIntConfig("watchdog-period", NULL, MODIFIABLE_CONFIG | HIDDEN_CONFIG, 0, INT_MAX, server.watchdog_period, 0, INTEGER_CONFIG, NULL, updateWatchdogPeriod),
     createIntConfig("shutdown-timeout", NULL, MODIFIABLE_CONFIG, 0, INT_MAX, server.shutdown_timeout, 10, INTEGER_CONFIG, NULL, NULL),
     createIntConfig("repl-diskless-sync-max-replicas", NULL, MODIFIABLE_CONFIG, 0, INT_MAX, server.repl_diskless_sync_max_replicas, 0, INTEGER_CONFIG, NULL, NULL),
+    createIntConfig("auth-threshold", NULL, MODIFIABLE_CONFIG, 0, 100, server.auth_threshold, 0, INTEGER_CONFIG, NULL, NULL),
+    createIntConfig("auth-delay", NULL, MODIFIABLE_CONFIG, 0, 120, server.auth_delay, 3, INTEGER_CONFIG, NULL, NULL),
```

Comment:
we need to agree on the defaults, if they're backwards compatible (feature is disabled), or set in some way.
arguably, even if the delay is disabled, we can still issue the log warning after a certain number of attempts so that fail2ban or alike can work with the default config.